### PR TITLE
Add support for is.past, is.future and is.today

### LINKED ISSUE #28: Add support for 'is.today'
Checks if the given date is today.

```js
is.today(new Date()); // true
```

**Update README.md**: under `is > DateTime`

### LINKED ISSUE #29: Add support for 'is.past'
Checks if the given date object is in the past.

**Update README.md**: under `is > DateTime`

### LINKED ISSUE #30: Add support for 'is.future'
Checks if the given date object is in the future.

**Update README.md**: under `is > DateTime`

### PR DESCRIPTION
**Pull request by boristane**
fixes #29 
fixes #30 
fixes #28

### CODE REVIEW

File: test/is/datetime.test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        expect(is.past(new Date('November 23, 3998 03:24:00'))).toBeFalsy();
+        expect(is.past('November 23, 3998 03:24:00')).toBeFalsy();
+        expect(is.past(new Date())).toBeFalsy();
+    });
+    testFalsyWithNullable(is.past);
+});
+
+describe('isFuture', () => {
+    test('returns true on future date objects', () => {
+        expect(is.future(new Date('November 23, 3998 03:24:00'))).toBeTruthy();
```
Comment (joeltankam) [inline test/is/datetime.test.js]: Test fail on `November 23, 3998 03:24:00`
I would have preferred something like : 
```js
var future = new Date.now();
future.setFullYear(Date.now().getFullYear() + 1);
...
```

Comment (boristane) [reply test/is/datetime.test.js]: Thanks for the feedback. Will update the test.

File: test/is/datetime.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+describe('isPast', () => {
+    test('returns true on past date objects', () => {
+        expect(is.past(new Date(0))).toBeTruthy();
+        expect(is.past(new Date(1000))).toBeTruthy();
+        expect(is.past(new Date('April 30, 1993 16:40:00'))).toBeTruthy();
+    });
+    test('returns false on anything else', () => {
+        expect(is.past(new Date('November 23, 3998 03:24:00'))).toBeFalsy();
+        expect(is.past('November 23, 3998 03:24:00')).toBeFalsy();
+        expect(is.past(new Date())).toBeFalsy();
```
Comment (joeltankam) [inline test/is/datetime.test.js]: Poorly designed.
Depends on the execution speed and the precision of the clock.
Please fix.

File: test/is/datetime.test.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+describe('isPast', () => {
+    test('returns true on past date objects', () => {
+        expect(is.past(new Date(0))).toBeTruthy();
+        expect(is.past(new Date(1000))).toBeTruthy();
+        expect(is.past(new Date('April 30, 1993 16:40:00'))).toBeTruthy();
+    });
+    test('returns false on anything else', () => {
+        expect(is.past(new Date('November 23, 3998 03:24:00'))).toBeFalsy();
+        expect(is.past('November 23, 3998 03:24:00')).toBeFalsy();
```
Comment (joeltankam) [inline test/is/datetime.test.js]: Same as the comment in`is.future` test

File: test/is/datetime.test.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        let date = new Date();
+        date.setHours(12);
+        date.setMinutes(30);
+        expect(is.today(new Date())).toBeTruthy();
+        expect(is.today(date)).toBeTruthy();
+    });
+    test('returns false on anything else', () => {
+        expect(is.today(new Date(0))).toBeFalsy();
+        expect(is.today(new Date('April 30, 1993 16:40:00'))).toBeFalsy();
+        expect(is.today('November 23, 3998 03:24:00')).toBeFalsy();
```
Comment (joeltankam) [inline test/is/datetime.test.js]: Fails on November 23, 3998

File: README.md [hunk 5]
```diff
@@ -292,20 +292,30 @@ Checks _if_ a given value _is_ a Date object in the past.
...
+futureDate.setUTCFullYear(futureDate.getUTCFullYear() + 1);
+is.future(futureDate); // true
+
 is.future(new Date(0)); // false
-is.future(new Date('April 30, 1993 16:40:00')); // false
-is.future(new Date()); // false
+
+let pastDate = new Date();
+pastDate.setUTCFullYear(pastDate.getUTCFullYear() - 1);
+is.past(pastDate); // false
```
Comment (joeltankam) [inline README.md]: Fix typo : `is.pass`

**Comment by joeltankam** [comment]
Thanks for your contribution 🎉

**Pull request merged**